Makefile

```diff
@@ -46,19 +34,24 @@ define etcd_component
 # python_component macro
 $1_SETUP = $1_setup.py shared_setup.py
 $1_REQUIREMENTS = $1-requirements.txt common/requirements.txt shared-requirements.txt
-$1_SOURCES = $(shell find src/metaswitch -type f -not -name "*.pyc") $(shell find common/metaswitch -type f -not -name "*.pyc")
+$1_TEST_REQUIREMENTS = common/requirements-test.txt requirements-test.txt
+$1_TEST_SETUP = $1_setup.py
+$1_SOURCES = $(shell find src/metaswitch -type f -not -name "*.pyc" -not -name "alarm_constants.py") $(shell find common/metaswitch -type f -not -name "*.pyc")
```

Comment:
Could we split this onto two lines?

What's the rationale behind alarm constants?

Shouldn't the sources for Python common be handled by metaswitchcommon?

Reply:
I've added a comment to explain the alarm constants. The common sources are probably handled by the common wheel, but our previous makefiles had this dependency and I didn't want to remove it in case it was needed for some reason.

I've not tried to split the long line because that made it less readable to my mind.